spec.html

```diff
@@ -25877,6 +26626,144 @@ <h1>Properties of Number Instances</h1>
     </emu-clause>
   </emu-clause>
 
+  <emu-clause id="sec-bigint-objects">
+  <h1>BigInt Objects</h1>
+    <emu-clause id="sec-bigint-constructor">
+      <h1>The BigInt Constructor</h1>
```

Comment:
Example of a new format to follow: https://tc39.github.io/ecma262/#sec-number-constructor

Reply:
Changed.